packages/app-treasury/src/Overview/Item.tsx

```diff
@@ -0,0 +1,61 @@
+// Copyright 2017-2019 @polkadot/app-democracy authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import { I18nProps } from '@polkadot/ui-app/types';
+
+import React from 'react';
+import styled from 'styled-components';
+import { TreasuryProposal } from '@polkadot/types';
+import { Card } from '@polkadot/ui-app';
+import { styles as rowStyles } from '@polkadot/ui-app/Row';
+
+import translate from '../translate';
+
+type Props = I18nProps & {
+  children?: React.ReactNode,
+  isApproved?: boolean,
+  proposal: TreasuryProposal,
+  proposalExtra?: React.ReactNode,
+  proposalId: string
+};
+
+class Item extends React.PureComponent<Props> {
+  render () {
+    const { children, className, proposalExtra, proposalId } = this.props;
+
+    return (
+      <Card className={className}>
+        <div className='ui--Row'>
```

Comment:
Yes, this is horrible. However, let's log it as an issue - we need to have some standard container we can use here and in democracy instead of doing what we are doing here (or what we are doing in democracy). Not really re-usable.

Reply:
Logged as https://github.com/polkadot-js/apps/issues/1336